src/theme/components/breadcrumb.ts

```diff
@@ -13,18 +16,39 @@ const blogs = {
 };
 const booksAndMore = {
   bg: "section.books-and-more.secondary",
+  _dark: {
+    bg: "dark.section.books-and-more.secondary",
+  },
+};
+const brand = {
+  bg: "brand.secondary",
+  _dark: {
+    bg: "dark.brand.secondary",
+  },
 };
 const education = {
   bg: "section.education.secondary",
+  _dark: {
+    bg: "dark.section.education.secondary",
+  },
 };
 const locations = {
   bg: "section.locations.primary",
+  _dark: {
+    bg: "dark.section.locations.primary",
+  },
 };
 const research = {
   bg: "section.research.secondary",
+  _dark: {
+    bg: "dark.section.research.secondary",
+  },
 };
 const whatsOn = {
   bg: "section.whats-on.secondary",
+  _dark: {
+    bg: "dark.section.whats-on.secondary",
```

Comment:
The function that is used to produce this value is returning an bad value because the root hex value for this color has 3 chars.  All the others have 6 chars and that is what the function is setup to handle.  To fix this, update the `hexToRGB` function (found in utils.ts) with the following:
```
export const hexToRGB = (hex: string, alpha: number) => {
  const shortHex = hex.length === 4;
  const rSlice = shortHex ? hex.slice(1, 2).repeat(2) : hex.slice(1, 3),
    gSlice = shortHex ? hex.slice(2, 3).repeat(2) : hex.slice(3, 5),
    bSlice = shortHex ? hex.slice(3, 4).repeat(2) : hex.slice(5, 7);
  const r = parseInt(rSlice, 16),
    g = parseInt(gSlice, 16),
    b = parseInt(bSlice, 16);
  const rgb = `${r}, ${g}, ${b}`;
  return alpha ? `rgba(${rgb},${alpha})` : `rgb(${rgb})`;
};
```